docs/how-to-generate-diagrams.md

```diff
@@ -0,0 +1,9 @@
+#How to generate diagrams
+
+Proxy diagrams should be generated by preference with Plantuml.
+Plantuml is a tool that allows to generate diagrams in different image formats
+using a text description.
+
+In you can generate diagrams by its web server: http://www.plantuml.com/plantuml/uml
```

Comment:
I'd make this fully impersonal. Something like, "Diagrams can be generated directly in a browser at http://www.plantuml.com/plantuml/uml, or locally at http://plantuml.com/download."